modules/unmanaged-operators.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture-installation.adoc
+// * updating/updating-cluster-between-minor.adoc
+
+[id="unmanaged-operators_{context}"]
+= Unmanaged Operators
+
+The _management state_ of an Operator determines whether an Operator is actively
+managing the resources for its related component in the cluster as designed. If
+an Operator is set to an unmanaged state, it does not respond to changes in
+configuration nor does it receive updates.
+
+While this can be helpful in non-production clusters or during debugging,
+Operators in an unmanaged state are unsupported and the cluster administrator
+assumes full control of the individual component configurations and upgrades.
+Depending on the method the Operator was set to unmanaged, it can also render
+the entire cluster unsupported.
+
+An Operator's management state is accessible using the following methods:
+
+* **Cluster Version Operator (CVO) overrides**
++
+The CVO configuration includes access to the `spec.overrides` parameter, which
+allows administrators to provide a list of overrides to the CVO's behavior for a
+component. For example:
++
+[source,yaml]
+----
+spec:
+  channel: stable-4.4
+  clusterID: 83062003-0ede-460e-9dea-3330cc8a37b3
+  overrides:
+  - group: apps/v1
+    kind: Deployment
+    name: network-operator
+    namespace: openshift-network-operator
+    unmanaged: true
+----
++
+The `spec.overrides[].unmanaged` parameter is set to `false` by default, however
+setting it to `true` blocks cluster upgrades and alerts the administrator that a
```

Comment:
Does this alert display during update or when you set the override?

Reply:
Seems like it's a status condition during update (https://bugzilla.redhat.com/show_bug.cgi?id=1822844#c0) as I couldn't find it when I only set an override, but trying again to check.

Reply:
OK now I saw it this time, it just takes a few minutes. Confirmed that `Upgradeable=false` and the error message are added shortly after you set the override.

Reply:
Should we show `unmanaged: false` in the code block as that is the default state? Presumably, if you are reading this for the first time and check  the CVO, it will say `false` (unless someone else changed it). 
Do we need a command to view/edit the CVO?

Reply:
@mburke5678 The whole `overrides` section actually isn't even there by default, the example snippet is just showing what it would look like it if it was added. I debated mentioning that too, might add.